03_qgep_db_dss.sql

```diff
@@ -3923,7 +3923,7 @@ ALTER TABLE qgep.vl_structure_part_renovation_demand ADD CONSTRAINT pkey_qgep_vl
  REFERENCES qgep.vl_structure_part_renovation_demand (code) MATCH SIMPLE 
  ON UPDATE RESTRICT ON DELETE RESTRICT;
 ALTER TABLE qgep.od_structure_part ADD COLUMN fk_wastewater_structure varchar (16);
-ALTER TABLE qgep.od_structure_part ADD CONSTRAINT rel_structure_part_wastewater_structure FOREIGN KEY (fk_wastewater_structure) REFERENCES qgep.od_wastewater_structure(obj_id);
+ALTER TABLE qgep.od_structure_part ADD CONSTRAINT rel_structure_part_wastewater_structure FOREIGN KEY (fk_wastewater_structure) REFERENCES qgep.od_wastewater_structure(obj_id) ON DELETE set null;
```

Comment:
Hi  @sjib 

Is the `ON DELETE SET NULL` action correct for the *structure part* -> *wastewater structure* relation?
I would expect that the structure parts are deleted as well when a wastewater structure is deleted. With `ON DELETE SET NULL` they will be orphaned.